core/src/main/scala/cromwell/core/WorkflowOptions.scala

```diff
@@ -42,10 +42,26 @@ import scala.util.{Failure, Success, Try}
  */
 
 object WorkflowOptions {
+
+  sealed trait WorkflowOption { def name: String }
```

Comment:
making this an abstract class that takes a constructor parameter `name` might be less noisy

```scala
case object WriteToCache extends WorkflowOption("write_to_cache")
```